_haikus/freshness.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: freshness
+author: tcd
+---
+
+the farm has freshness
+both on the ground and way up high
```

Comment:
This line has one too many syllables, consider this change:

```
 both on the ground and up high<br>
```

Reply:
Ahh, that one slipped by me. I think that change works perfectly.